dvc/repo/checkout.py

```diff
@@ -87,9 +95,7 @@ def _checkout(
                 filter_info=filter_info,
             )
             for key, items in result.items():
-                stats[key].extend(
-                    _fspath_dir(path, self.root_dir) for path in items
-                )
+                stats[key].extend(_fspath_dir(path) for path in items)
```

Comment:
Before, the checkout summary were shown as `path_in_repo`.  This should now actually show in terms of relpath.